src/web/middleware/mod.rs

```diff
@@ -0,0 +1,40 @@
+pub mod db;
+pub mod precondition;
+pub mod sentry;
+pub mod weave;
+
+// # Web Middleware
+//
+// Matches the [Sync Storage middleware](https://github.com/mozilla-services/server-syncstorage/blob/master/syncstorage/tweens.py) (tweens).
+
+use actix_web::dev::ServiceRequest;
+use actix_web::Error;
+
+use crate::db::util::SyncTimestamp;
+use crate::error::{ApiError, ApiErrorKind};
+use crate::server::ServerState;
+use crate::web::{extractors::HawkIdentifier, tags::Tags, DOCKER_FLOW_ENDPOINTS};
+
+/// The resource in question's Timestamp
+pub struct ResourceTimestamp(SyncTimestamp);
```

Comment:
apparently this is dead code, we can remove it later

Reply:
Huh, wonder why it wasn't getting flagged? Guessing it's because it's in a `mod.rs` file. Good to know for the future.